src/engraving/devtools/drawdata/drawdatagenerator.cpp

```diff
@@ -101,7 +101,7 @@ Ret DrawDataGenerator::processFile(const muse::io::path_t& scoreFile, const muse
 
 DrawDataPtr DrawDataGenerator::genDrawData(const muse::io::path_t& scorePath, const GenOpt& opt) const
 {
-    MasterScore* score = compat::ScoreAccess::createMasterScoreWithBaseStyle();
+    MasterScore* score = compat::ScoreAccess::createMasterScoreWithBaseStyle(nullptr);
```

Comment:
What is the consequence of passing `nullptr` here? Is it that this score will not be able to find `EngravingConfiguration` and `Renderer`? (I _guess_ that's fine, until the moment that it isn't fine anymore because it causes crashes, but just wanted to check)

Reply:
Now it is done so that if a ioc context is null, then the global ioc is used. It might be worth doing this more explicitly, for example passing "global ioc context"

Reply:
But not all interfaces are registered to the global ioc context, right? So those that aren't registered there, those will not be found here.

Reply:
All interfaces are registered in the global ioc, because the “global ioc” will simply be the first ioc (of the first window).